docs/content/docs/testing/typed-telemetry.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Typed telemetry
+menuPosition: 3
```

Comment:
I think it might make sense to make this #2 and the existing telemetry one #3, but I need to chew on it more. Might also consider a bit of updating the existing one. There's good content there, but I think it might be confusing to have both articles especially if that one doesn't say something along the lines of "this is for Fluid's internal telemetry which is subject to change", and a reworded title.